core/utils/poly-look/package.json

```diff
@@ -13,6 +13,7 @@
     "test:watch": "web-test-runner --watch",
     "storybook": "web-dev-server -c .storybook/server.mjs",
     "build": "rollup -c rollup.config.js",
+    "win-build": "rollup -c rollup.config.js",
```

Comment:
Ha! That's the kind of stuff I mean, if `build.js` would fall back to the `build` script if there is no `win-build` script, we wouldn't have to duplicate things. (Although you could also avoid duplication locally by just calling the `build` script from `win-build`, but that still means we need to add boilerplate to every package that's easy to forget).